### ISSUE TITLE
Accessibility  - not accessible by keyboard alone

### ISSUE THREAD

**Issue by hananamar** [AddToCalendar/addtocalendar#44]
Hi there,
I ran into an accessibility issue.

On the one hand, plugin sets TabIndex on the main button to "1" (wish I could be able to change this via a var, but never mind). So one could navigate to the main button via the tab button.

But on the other hand using the tab button does not allow using any of the action links in the drop down. The drop down menu of actions disappears as soon as the tab button is hit again.

Hope this can be fixed easily,
Cheers!

**Comment by christele15** [comment]
I HAVE EXACTLY THE SAME ISSUE!!! I have been pulling my hair trying to fix this !!!

**Comment by grove86nickson** [comment]
Accessibility issues are important.I am facing the same issue.Please prioritize.
Remove positive tabindex
